### PR TITLE
Add new plugin in the visualization board

### LINKED ISSUE #4: Open maps from visualize
Add support to open Maps from Visualize → Create New Visualization → Maps

### PR DESCRIPTION
**Pull request by DDG18**
### Description
Define createMapsExplorerDashboardsVisTypeDefinition()
Leave all config in createMapsExplorerDashboardsVisTypeDefinition empty
Screen Shot 2022-05-17 at 5 56 05 PM

 
### Issues Resolved
#4 
 
### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/to_ast.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
```
Comment (VijayanB) [inline public/to_ast.ts:32]:
```suggestion
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */
```

File: public/to_ast.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  const mapsExplorer = buildExpressionFunction<MapsExplorerDashboardsExpressionFunctionDefinition>('maps_explorer_dashboards_vis', {
+    expression,
+    interval,
+  });
+
+  const ast = buildExpression([mapsExplorer]);
+
+  return ast.toAst();
+};
```
Comment (VijayanB) [inline public/to_ast.ts:48]: Lets have an empty line as EOF

File: public/types.ts [hunk 3]
```diff
@@ -9,3 +13,40 @@ export interface MapsExplorerDashboardsPluginStart {}
...
+  Render<MapsExplorerDashboardsRenderValue>
+>;
+
+export const createMapsExplorerDashboardsVisTypeDefinition = (): BaseVisTypeOptions => ({
+  name: 'maps_explorer_dashboards',
+  // TODO: Define String ID  'visTypeMetric.mapsExplorerDashboardsTitle'
+  title: i18n.translate('visTypeMetric.mapsExplorerDashboardsTitle', { defaultMessage: 'Maps Explorer' }),
+  // TODO: change to the unique icon later
+  icon: 'visMapRegion',
+  // TODO: Define String ID 'visTypeMetric.metricDescription'
```
Comment (VijayanB) [inline public/types.ts:45]: Let's create an issue for TODO

Comment (DDG18) [reply public/types.ts:45]: #6

File: public/to_ast.ts (hunk 2 above)
Comment (DDG18) [reply public/to_ast.ts:48]: Updated

File: public/to_ast.ts (hunk 1 above)
Comment (DDG18) [reply public/to_ast.ts:32]: Edited

**Review by VijayanB — APPROVED** [review]

**Pull request merged**